### PR TITLE
docs: added initial documentation files

### PR DESCRIPTION
**Pull request by shmorri**

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Project information
+site_name: Gluu Flex 1.0 Docs
```
Comment (moabu) [inline docs/mkdocs.yml:2]: Continuing on the versioning to fit with services like Casa and to probably make more sense moving from 4.x perhaps this should be 5.0 ?

File: docs/mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+site_description: Documentation for Gluu Flex
+
+# Copyright
+copyright: Copyright &copy; 2022, Gluu, Inc.
+
+# Repository
+docs_dir: source
+repo_name: GluuFederation/flex
+repo_url: https://github.com/GluuFederation/flex/
+edit_uri: edit/4.3/docs/source
```
Comment (moabu) [inline docs/mkdocs.yml:12]: this should be modified. The source should be directing to the docs folder inside the repo

File: docs/mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+theme:
+    name: material
+    custom_dir: 'theme'
+    palette:
+        primary: 'green'
+        accent: 'green'
+    logo: 'gluu.jpg'
+    favicon: 'favicon.ico'
+extra:
+    version: '1.0'
```
Comment (moabu) [inline docs/mkdocs.yml:24]: versioning ? Same as point 1

**Review by moabu — CHANGES_REQUESTED** [review]

File: docs/mkdocs.yml (hunk 1 above)
Comment (shmorri) [reply docs/mkdocs.yml]: Corrected to 5.0

File: docs/mkdocs.yml (hunk 2 above)
Comment (shmorri) [reply docs/mkdocs.yml]: I did correct it to the `main` branch. The `source` folder is inside the docs folder inside the repo. This is standard for Mkdocs, the pathing is relative to mkdocs.yml. The `source` folder, as well as an optional `theme` override folder go here, which Mkdocs pulls the .md files from.

**Pull request merged**